src/Umbraco.Web.UI.NetCore/Startup.cs

```diff
@@ -47,9 +47,6 @@ public void ConfigureServices(IServiceCollection services)
             services.AddUmbracoRuntimeMinifier(_config);
             services.AddUmbracoBackOffice();
             services.AddUmbracoBackOfficeIdentity();
-
-            services.AddMvc();
```

Comment:
See my other note, i'm unsure we want to move this right now

Reply:
I understand, but I don't what to add a back office requirement to the startup. So it must be in one of our extension methods..